app/gas/views.py

```diff
@@ -25,29 +26,22 @@
 from django.utils.translation import gettext_lazy as _
 
 from cacheops import CacheMiss, cache, cached_view, cached_view_as
+from dashboard.helpers import handle_bounty_views
 from economy.utils import convert_amount
-from gas.models import GasGuzzler
-from gas.utils import conf_time_spread, gas_advisories, gas_history, recommend_min_gas_price_to_confirm_in_time
 from perftools.models import JSONStore
 
-from .helpers import handle_bounty_views
+from .models import GasGuzzler
+from .utils import (conf_time_spread, gas_advisories, gas_history,
+                    recommend_min_gas_price_to_confirm_in_time)
 
 logging.basicConfig(level=logging.DEBUG)
 
 confirm_time_minutes_target = 4
 
-lines = {
-    1: 'red',
-    5: 'orange',
-    60: 'green',
-    90: 'steelblue',
-    105: 'purple',
-    120: '#dddddd',
-    180: 'black',
-}
+lines = {1: 'red', 5: 'orange', 60: 'green', 90: 'steelblue', 105: 'purple', 120: '#dddddd', 180: 'black', }
```

Comment:
IMHO the old way is easier to read / mutate

Reply:
@owocki 
Gotcha - I hadn't realized we regularly changed these values.  I'll go ahead and switch it back.

Some considerations moving forward:

- we can split this variable off into a constants module and treat it as a module level variable
- capitalized naming convention
- all hex codes or defined color names for consistency
- validating modifications to this dict if it regularly changes via tests and/or dynamically store them somewhere (probably not the most favorable option)